doc/developer_guide/developer_environment.md

```diff
@@ -0,0 +1,50 @@
+# Developer Environment
+
+## Installation
+
+### Project Requirements
+
+For this project you need
+
+- Lua 5.4
+- Luarocks 3.*
+- Python 3.8
+- Poetry
+
+### Install on a desktop machine
+
+1. Install poetry following the instructions on their [website](https://python-poetry.org/docs/).
+2. Install micromamba as explained on their [website](https://mamba.readthedocs.io/en/latest/installation.html)
```

Comment:
wouldn't it work with conda, too?

Reply:
I saw the bash scripts use mamba. So probably it will not work with conda.
But I saw there is an install script for mamba, is it worth to mention that one here?

Reply:
Installing micromamba is a much less a hassle, but yes in theory you could replace micromamba with conda.